File: go/international/09_get_relationships.go
package main

import (
	"encoding/json"
	"fmt"
)

func main() {
	baseURL := "https://sandbox.easylink.id"
	appID := "lQNJ0nL07Ucmemaa"
	appSecret := "HrfFeuRmoyBsZhxDi3w3JNdxwYu19lL4"
	appKey := "3f9a7f74-de23-4fde-af75-da7684528a59"
	privateKeyPem := "../../private_key.pem"

	accessToken, err := GetAccessToken(baseURL, appID, appSecret)
	if err != nil {
		fmt.Printf("Get token error: %v\n", err)
		return
	}

	payload := map[string]interface{}{
		"segment": "business",
	}

	fmt.Println("Fetching Relationships...")
	_, res, err := SendEasylinkRequest(baseURL, "/data/get-relationships", "POST", payload, appKey, privateKeyPem, accessToken)
	if err != nil {
		fmt.Printf("Error: %v\n", err)
	} else {
		resBytes, _ := json.MarshalIndent(res, "", "  ")
		fmt.Printf("Response:\n%s\n", string(resBytes))
	}
}

File: python/international/09_get_relationships.py
import sys
import json
from pathlib import Path
sys.path.append(str(Path(__file__).resolve().parent.parent))

from helpers import send_easylink_request, get_access_token

base_url        = 'https://sandbox.easylink.id'
app_id          = 'lQNJ0nL07Ucmemaa'
app_secret      = 'HrfFeuRmoyBsZhxDi3w3JNdxwYu19lL4'
app_key         = '3f9a7f74-de23-4fde-af75-da7684528a59'
private_key_pem = str(Path(__file__).resolve().parent.parent.parent / 'private_key.pem')

payload = {
    'segment': 'business'
}

print("Fetching Relationships...")
try:
    access_token = get_access_token(base_url, app_id, app_secret)
    res = send_easylink_request(
        base_url,
        '/data/get-relationships',
        'POST',
        payload,
        app_key,
        private_key_pem,
        access_token
    )
    print("Response:\n", json.dumps(res['data'], indent=2))
except Exception as e:
    print("Error:", e)
